src/uct/rocm/cma/rocm_common.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) Advanced Micro Devices, Inc. 2016 - 2017. ALL RIGHTS RESERVED.
+ * Copyright (C) Mellanox Technologies Ltd. 2019.       ALL RIGHTS RESERVED.
```

Comment:
Why so many whitespaces? In all files we have only 2 :)

Reply:
single year, later will add 2019-....

Reply:
anyway it looks strange